website/cue/reference/components/base/sources.cue

```diff
@@ -47,11 +47,11 @@ base: components: sources: configuration: proxy: {
 
 				| Pattern             | Example match                                                               |
 				| ------------------- | --------------------------------------------------------------------------- |
-				| Domain names        | `**example.com**` matches requests to `**example.com**`                     |
-				| Wildcard domains    | `**.example.com**` matches requests to `**example.com**` and its subdomains |
-				| IP addresses        | `**127.0.0.1**` matches requests to `**127.0.0.1**`                         |
-				| [CIDR][cidr] blocks | `**192.168.0.0/16**` matches requests to any IP addresses in this range     |
-				| Splat               | `__*__` matches all hosts                                                   |
+				| Domain names        | `example.com` matches requests to `example.com`                     |
+				| Wildcard domains    | `.example.com` matches requests to `example.com` and its subdomains |
+				| IP addresses        | `127.0.0.1` matches requests to `127.0.0.1`                         |
+				| [CIDR][cidr] blocks | `192.168.0.0/16` matches requests to any IP addresses in this range |
+				| Splat               | `*` matches all hosts                                               |
```

Comment:
We'll want to actually make the changes [here](https://github.com/vectordotdev/vector/blob/cf7372979be3b818f1cece9dae286a50a1554bf9/lib/vector-core/src/config/proxy.rs#L72-L84) so when we regenerate the cue it'll be correct still.

Reply:
Thanks @spencergilbert!  Will do.